rules/busybox.bzl

```diff
@@ -118,6 +118,10 @@ def _make_resources_flag(
         ],
     )
 
+def _disable_warnings(args):
+    # Disable warnings - this are output to stdin/stderr which breaks worker mode
+    args.add("--logWarnings=false")
```

Comment:
What happens to warnings in worker mode? I think we'd still want to know about them. Would they get dumped to a non-stdout/stderr file?

Reply:
It looks like this is just mirroring the existing behavior inside `BusyBoxActionBuilder` https://github.com/bazelbuild/bazel/blob/84c1ed430405b154b6e9eb2c28281f450e250eff/src/main/java/com/google/devtools/build/lib/rules/android/BusyBoxActionBuilder.java#L369